Tasks/AzureRmWebAppDeployment/VariableSubstituter.ps1

```diff
@@ -0,0 +1,85 @@
+﻿function Substitute-Variables
+{  
+    Param(
+        [string][Parameter(mandatory=$true)] 
+        $WebAppFolderPath,
+        [string][Parameter(mandatory=$true)] 
+        $ConfigFileRegex
+    )
+
+    # Load modules
+    . $PSScriptRoot/XMLSubstituter.ps1 -Force
+
+    # Defined supported MIME type
+    $ApplicationXmlMimeType = "application/xml"
+    $TextXmlMimeType = "application/xml"
+    $ConfigExtension = ".config"
+
+    # Find all configuration files
+    $webConfigFiles = Get-ChildItem -Path $WebAppFolderPath -Filter $ConfigFileRegex -Recurse -ErrorAction SilentlyContinue -Force
```

Comment:
Find-VstsFiles also gives this functionality, may be it has some additional checks and balances. Do you think it is better to use that?